addons/api/mirage/config.js

```diff
@@ -270,26 +270,7 @@ function routes() {
   });
 
   // IAM: Groups
-  this.get(
-    '/groups',
-    ({ groups }, { queryParams: { scope_id, recursive, filter } }) => {
-      let resultSet;
-      if (recursive && scope_id === 'global') {
-        resultSet = groups.all();
-      } else if (recursive) {
-        resultSet = groups.where((group) => {
-          const groupModel = groups.find(group.id);
-          return (
-            group.scopeId === scope_id ||
-            groupModel?.scope?.scope?.id === scope_id
-          );
-        });
-      } else {
-        resultSet = groups.where((group) => group.scopeId === scope_id);
-      }
-      return resultSet.filter(makeBooleanFilter(filter));
-    },
```

Comment:
I wasn't suggesting we keep it, it was a genuine question if we needed it anymore. I assume since everything is doing a global recursive call that it doesn't get used anymore but just wanted to confirm that